src/components/AlertDialog.tsx

```diff
@@ -1,62 +1,41 @@
-import React, { FC, useState } from 'react';
+import React, { FC } from 'react';
 import Button from '@mui/material/Button';
 import Dialog from '@mui/material/Dialog';
 import DialogActions from '@mui/material/DialogActions';
 import DialogContent from '@mui/material/DialogContent';
 import DialogContentText from '@mui/material/DialogContentText';
 import DialogTitle from '@mui/material/DialogTitle';
+import { DialogConfiguration } from 'context/DialogContext';
 
-type AlertDialogProps = {
-    handleDelete: () => void;
-};
-
-const AlertDialog: FC<AlertDialogProps> = ({ handleDelete }) => {
-    const [open, setOpen] = useState(false);
+interface AlertDialogProps extends DialogConfiguration {
+    open: boolean;
+    onClose: () => void;
+}
 
-    const handleClickOpen = () => {
-        setOpen(true);
-    };
 
-    const handleClose = () => {
-        setOpen(false);
-    };
+export const AlertDialog: FC<AlertDialogProps> = ({ open, title, variant, description, onSubmit, onClose }) => {
 
-    const handleConfirm = () => {
-        handleDelete();
-        setOpen(false);
-    };
+    const handleDelete = () => {
```

Comment:
nie bez powodu to sie tak nazywalo. `handleDelete` informuje, ze ten dialog sluzy tylko do usuwania- a tak nie jest, jest reuzywalny. Najlepiej byloby to nazwac `handleSubmit`

Reply:
nie wiem czy dobrze zrozumiałem ale nazwa zmieniona. wszytsko w nowym commicie